istio.VERSION

```diff
@@ -2,8 +2,8 @@
 # install/updateVersion.sh (see install/README.md)
 export CA_HUB="docker.io/istio"
 export CA_TAG="0.1.5"
-export MIXER_HUB="docker.io/istio"
-export MIXER_TAG="0.2.0-82070e1"
-export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/06ee3635b2f10716abe3d59f625e0000649ccc59/artifacts/istioctl"
+export MIXER_HUB="gcr.io/istio-testing"
```

Comment:
try use this. note it's istio-test not testing
--mixer_hub=gcr.io/istio-test
--mixer_tag=xiaolan2